storybook/storybook-react/src/OrderedList/OrderedList.stories.tsx

```diff
@@ -7,19 +7,29 @@ import { Heading, OrderedList, Paragraph } from '@amsterdam/design-system-react'
 import type { Meta, StoryObj } from '@storybook/react'
 import { exampleOrderedList } from '../shared/exampleContent'
 
-const orderedList = exampleOrderedList().map((text, index) => <OrderedList.Item key={index}>{text}</OrderedList.Item>)
+const orderedListItems = exampleOrderedList().map((text, index) => (
+  <OrderedList.Item key={index}>{text}</OrderedList.Item>
+))
 
 const meta = {
   title: 'Text/Ordered List',
   component: OrderedList,
   args: {
-    children: orderedList,
-    inverseColor: undefined,
+    children: orderedListItems,
+    inverseColor: false,
     markers: undefined,
   },
   argTypes: {
+    inverseColor: { control: 'boolean' },
     markers: { control: 'boolean' },
```

Comment:
Do we want to add `start` and `reversed` here as well? That would unify all controls for all stories